_content/av/silent-illumination_david-listen.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Silent Illumination"
+authors:
+  - "David Listen"
+drive_links:
+  - "https://drive.google.com/file/d/17YkFu2aSk-nqtpCWT3HIO6Yi0bAWAK6A/view?usp=drive_link"
```

Comment:
Oh! Interesting. This was ripped from a YouTube video which has apparently since been deleted. This is why I back stuff up to Google Drive!